doc/dev/encoding.rst

```diff
@@ -55,7 +55,7 @@ of interoperability between dencoders:
 
 * ``n-2`` for dencoding between daemons,
 * ``n-3`` hard requirement for client-involved scenarios,
-* ``n-3..``  soft requirements for clinet-involved scenarios. Ideally
+* ``n-3..``  soft requirements for client-involved scenarios. Ideally
```

Comment:
Suggest just "for client scenarios"

Reply:
I'll pick this up in a future PR.

Reply:
https://github.com/ceph/ceph/pull/58451 makes this change, as well as a number of other minor changes.